### ISSUE TITLE
v1

### ISSUE THREAD

**Issue by joshgillies** [joshgillies/node-matrix-assets#2]
I'm almost happy with the API here, a few more tests and documentation should be written before releasing this as v1.

Regarding features currently I feel as though pretty much everything is covered, however it may be worth expanding on [`joshgillies/zion-assets`](https://github.com/joshgillies/zion-assets) a bit more to see if there are any gaps.

**Comment by joshgillies** [comment]
Preliminary integration of this module is also being conducted over at [`joshgillies/node-matrix-assets-xml`](https://github.com/joshgillies/node-matrix-assets-xml). Issues and missing features are being teased out as a result.

**Comment by joshgillies** [comment]
Misleading version is misleading. Although this is technically a v1 release (6f829da4caf916a40f9a8d5077af48a62a37f4a6), you'll note that we've published it as v3?! This is a work around for the fact that `node-matrix-assets@"<3.0.0"` was actually the foundation for what became `node-matrix-asset-list`. With v3 published I've now deprecated `node-matrix-assets@"<3.0.0"`, prompting users to migrate over to `node-matrix-asset-list`.
